_episodes/03-packaging-installing.md

```diff
@@ -116,129 +117,133 @@ This shows the basic commands and the general options.
 
-Making python code a package (packaging it) requires no extra tools. We need to:
+Making python code into a package requires no extra tools. We need to
 
-- Create a directory and give it package's name.
-- Put modules in it.
+- Create a directory, named after our package.
+- Put modules (`.py` files) in the directory.
 - Create an `__init__.py` file in the directory
-- Create a setup.py file at the top level
+- Create a `setup.py` file alongside the directory
 
-The `__init__.py` file tells python that the directory is supposed to be read as a package.
+Our final package will look like this:
 
-e.g.
+├── package-name  
+│   ├── \_\_init__.py  
+│   ├── module-a.py  
+│   └── module-b.py  
+└── setup.py
 
-Let us create a package called **Vehicles** with two modules **Land** and **Water**.
+The `__init__.py` file tells python that the directory is supposed to be tread as a package.
+
+Let's create a package called **conversions** with two modules **temp** and **speed**.
 
 ### Step 1: Creating a directory
-Create a directory called **Vehicles**
+Create a directory called **conversions**
 
 ~~~
-mkdir Vehicles
+mkdir conversions
 ~~~
 {: .language-bash}
 
 ### Step 2: Adding Modules
 
+conversions/temp.py
 ~~~
-class Land:
-    def __init__(self):
-        ''' Constructor for this class. '''
-        # Create some vehicles
-        self.members = ['Cars', 'Trucks']
-
-    def printMembers(self):
-        print('Printing members of the Land  class')
-        for member in self.members:
-            print('\t%s ' % member)
+def fahr_to_celsius(temp):
+    return ((temp - 32) * (5/9))
 ~~~
 {: .language-python}
 
-The class has a property named members – which is a list of some vehicles we might be interested in. It also has a method named printMembers which simply prints the list of vehicles of this class.Note: all classes defined must be capable of being imported, and won't be executed directly. this will be saved in a file called **Land.py**
+the file temp.py will be treated as a module called temp. This module contains the function `fahr_to_celsius`. The top level container is the package `conversions`. The end user will import this as:
+`from conversions.temp import fahr_to_celsius` 
 
-Next we create another module named **Water**. Create a file named **Water.py** inside the **Vehicles** directory and add the code below:
 
-~~~
-def isWater(v):
-    waterv =['ship', 'boat']
-    for obj in v:
-        if obj in waterv:
-            print("It is a water vehicle")
-        else:
-            print("It is not a water vehicle")
-~~~
-{: .language-python}
+> ## Exercise
+> 1. Create a file named **speed.py** inside the **conversions** directory and add a function named `kph_to_ms` that will convert kilometres per hour to meters per second:
+>
+> > ## Solution
+> > conversions/speed.py
+> > ~~~
+> > def kph_to_ms(speed):
+> >     return speed / 3.6
+> > ~~~
+> > {: .language-python}
+> {: .solution}
+{: .challenge}
```

Comment:
lets have the example docstring for the function in the exercise so that it's the example?

Reply:
I've added docstrings to the temperature and speed functions in 69eec4e